specs/wp-log-in-out-spec.js

```diff
@@ -110,302 +109,336 @@ describe( `[${ host }] Authentication: (${ screenSize }) @parallel @jetpack @vis
-	// 		before( function() {
-	// 			return driverManager.clearCookiesAndDeleteLocalStorage( driver );
-	// 		} );
-	//
-	// 		describe( 'Can request a magic link email by entering the email of an account which does not have a password defined', function() {
-	// 			let magicLoginLink, loginFlow, magicLinkEmail, emailClient;
-	// 			before( function() {
-	// 				loginFlow = new LoginFlow( driver, [ '+passwordless', '+2fa-otp' ] );
-	// 				emailClient = new EmailClient( get( loginFlow.account, 'mailosaur.inboxId' ) );
-	// 				return loginFlow.login();
-	// 			} );
-	//
-	// 			step( 'Can find the magic link in the email received', function() {
-	// 				return emailClient.pollEmailsByRecipient( loginFlow.account.email ).then( function( emails ) {
-	// 					magicLinkEmail = emails.find( email => email.subject.indexOf( 'WordPress.com' ) > -1 );
-	// 					assert( magicLinkEmail !== undefined, 'Could not find the magic login email' );
-	// 					magicLoginLink = magicLinkEmail.html.links[0].href;
-	// 					assert( magicLoginLink !== undefined, 'Could not locate the magic login link in the email' );
-	// 					return true;
-	// 				} );
-	// 			} );
-	//
-	// 			describe( 'Can use the magic link and the code received via sms to log in', function() {
-	// 				let magicLoginPage, twoFALoginPage;
-	// 				before( function() {
-	// 					driver.get( magicLoginLink );
-	// 					magicLoginPage = new MagicLoginPage( driver );
-	// 					magicLoginPage.finishLogin();
-	// 					twoFALoginPage = new LoginPage( driver );
-	// 					return twoFALoginPage.use2FAMethod( 'otp' );
-	// 				} );
-	//
-	// 				step( 'Should be on the /log-in/authenticator page', function() {
-	// 					return twoFALoginPage.displayed().then( function() {
-	// 						return driver.getCurrentUrl().then( ( urlDisplayed ) => {
-	// 							assert( urlDisplayed.indexOf( '/log-in/authenticator' ) !== -1, 'The 2fa authenticator page is not displayed after log in' );
-	// 						} );
-	// 					} );
-	// 				} );
-	//
-	// 				step( 'Enter the 2fa code and we\'re logged in', function() {
-	// 					const twoFACode = speakeasy.totp( {
-	// 						secret: loginFlow.account['2faOTPsecret'],
-	// 						encoding: 'base32'
-	// 					} );
-	// 					return twoFALoginPage.enter2FACode( twoFACode );
-	// 				} );
-	//
-	// 				// we should always remove a magic link email once the magic link has been used (even if login failed)
-	// 				after( function() {
-	// 					if ( magicLinkEmail ) {
-	// 						return emailClient.deleteAllEmailByID( magicLinkEmail.id );
-	// 					}
-	// 				} );
-	// 			} );
-	//
-	// 			after( function() {
-	// 				if ( loginFlow ) {
-	// 					loginFlow.end();
-	// 				}
-	// 			} );
-	// 		} );
-	// 	} );
-	// }
+	if (
+		dataHelper.hasAccountWithFeatures( '+2fa-sms -passwordless' ) &&
+		! dataHelper.isRunningOnLiveBranch()
+	) {
+		describe( 'Can Log in on a 2fa account', function() {
+			let loginFlow, twoFALoginPage, twoFACode;
```

Comment:
Tests are failing for me locally on this step - text set in input field is `1234567undefined` and because of that test cannot proceed.